client/main.scss

```diff
@@ -51,6 +51,9 @@ html, body, #react-root, [data-reactroot] {
     position: relative;
     overflow-y: auto;
     flex: 1 1;
+    .container {
+      padding-bottom: 1rem;
+    }
```

Comment:
Can we do this with bootstrap class `p-b-1` instead?  I think it's better than more noise in the SCSS.